website/docs/r/organization_project.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "github"
+page_title: "GitHub: github_organization_project"
+sidebar_current: "docs-github-resource-organization-project"
+description: |-
+  Creates and manages projects for Github organizations
+---
+
+# github_organization_project
+
+This resource allows you to create and manage projects for Github organization.
+
+## Example Usage
+
+```hcl
+resource "github_organization_project" "project" {
+  name = "A Organization Project"
+  body = "This is a organization project."
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - The name of the project.
+
+* `body` - The body of the project.
```

Comment:
Can you please document whether each field is `Optional` or `Required` here?

Reply:
Added in 2ea7426. Thanks!